inc/xen-version.inc

```diff
@@ -3,6 +3,6 @@
 # Following inc file defines XEN version and needed glue info to get it
 # built with the current yocto version
 ################################################################################
-require ../meta-xt-images-domx/recipes-extended/xen/xen-4.13-thud.inc
+require ../meta-xt-images-domx/recipes-extended/xen/xen-4.14-thud.inc
```

Comment:
@iusyk this is the dependency on the other external layer.
Just imagine that we have meta-xt-images-domx not at "../" location, but somewhere else and that will break
Beside that using an inc file from the other meta-layer looks odd.

Reply:
I am working with this

Reply:
> @iusyk this is the dependency on the other external layer.
> Just imagine that we have meta-xt-images-domx not at "../" location, but somewhere else and that will break
> Beside that using an inc file from the other meta-layer looks odd.

Frankly, I can not understand why xen-4.14-thud.inc belongs to meta-xt-images - this file is not used there. I think it is reasonable move this file in product. Any objections ?

Reply:
many things in meta-xt-images are not used by it directly, it is intended to contain common for all products stuff, I think.